code/contacts/org.eclipse.scout.contacts/pom.xml

```diff
@@ -23,7 +23,7 @@
 
   <properties>
     <jdk.source.version>1.8</jdk.source.version>
-    <org.eclipse.scout.rt.version>9.0.0-SNAPSHOT</org.eclipse.scout.rt.version>
```

Comment:
What is the idea here? We always use snapshot in the original source. The build will replace it with the actual version.

Reply:
Ok.